src/components/Section/Identification/IdentificationConnector.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import {
+  updateApplication,
+  reportErrors,
+} from '../../../actions/ApplicationActions'
+
+const connectIdentificationSection = (Component, { section, subsection, store, storeKey }) => {
```

Comment:
I wrote this HOC just for the Identification section, however it could definitely be abstracted and applied to any section.